web/themes/joinup/joinup_theme.theme

```diff
@@ -8,26 +8,37 @@
 use Drupal\Core\Template\Attribute;
 use Drupal\file\Entity\File;
 use Drupal\image\Entity\ImageStyle;
+use Drupal\Component\Utility\Html;
+use Drupal\Component\Utility\Unicode;
 
 /**
  * Implements hook_preprocess_page().
  */
 function joinup_theme_preprocess_page(&$variables) {
-  // Set page title.
   $request = \Drupal::request();
   $route_match = \Drupal::routeMatch();
+
+  // Set page title.
   $title = \Drupal::service('title_resolver')->getTitle($request, $route_match->getRouteObject());
-  $variables['title'] = $title;
 
   $route_name = $route_match->getRouteName();
   $blue_header_routes = array(
     'user.login',
     'user.register',
     'user.pass',
+    'node.add',
+    'node.add_page',
+    'rdf_entity.rdf_add',
+    'entity.node.edit_form',
+    'collection.propose_form',
+    'rdf_entity.rdf_add_page',
+    'entity.rdf_entity.edit_form',
   );
 
   if (!empty($route_name) && in_array($route_name, $blue_header_routes)) {
     $variables['is_blue_header'] = TRUE;
+    $title = $variables['page']['content']['pagetitle'];
```

Comment:
Again here, something like
`// Move the title from the content region to {...}`.